admin-apis/createApp.js

```diff
@@ -0,0 +1,49 @@
+import axios from "axios";
+import { generateApiSignature } from "./utils.js";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+// Function to create a new app using the account key
+const createNewAppUsingAccountKey = async () => {
```

Comment:
Remove UsingAccountKey from all functions. This directory only have admin-api code so it's not needed.

Reply:
done